tools/code-owners-parser/CodeOwnersParser/CodeownersFile.cs

```diff
@@ -38,19 +36,17 @@ public static List<CodeownersEntry> GetCodeownersEntries(string codeownersConten
 
         public static CodeownersEntry GetMatchingCodeownersEntry(
             string targetPath,
-            string codeownersFilePathOrUrl,
-            bool useRegexMatcher = UseRegexMatcherDefault)
+            string codeownersFilePathOrUrl)
```

Comment:
@JimSuplizio FYI if you are passing this third parameter and not using the default this will break when you get this update.